### DISCUSSION TITLE
Custom link: if statement with list

### DISCUSSION THREAD

**Discussion by tux-mania** [netbox-community/netbox#9535] [Q&A]
Hi! I'm trying to create a custom link which checks the circuit type and status.
I currently have this:
```
{% if obj.circuit_terminations.filter(circuit__custom_field_data__WAN='1', circuit__type='3', circuit__status='active').count() > 0%}T1
{% elif obj.circuit_terminations.filter(circuit__custom_field_data__WAN='1', circuit__type='4', circuit__status='active').count() > 0%}T1
{% elif obj.circuit_terminations.filter(circuit__custom_field_data__WAN='1', circuit__type='5', circuit__status='active').count() > 0%}T2
{% endif %}
```
This code actually works. The thing is, since I have multiple lines to write on the custom link and these ones are limited, I need to check (in this example) if 'circuit__type' has one of the following values: '3', '4'.

What syntax should I use in order to do it, like I would do in python:
```
if circuit_type in ['3', '4']:
```

Thank you for your help.

**Comment by hagbarddenstore** [comment]
circuit__type__in=[’3’, ’4’]

**Reply by tux-mania** [reply]
Hi. Thanks for your help.
Unfortunately, when trying with `circuit__type__in=[’3’, ’4’]` , I get the following error: `circuit__type__in is undefined`
